include/bluetooth/conn.h

```diff
@@ -303,10 +303,13 @@ struct bt_conn_br_info {
 	const bt_addr_t *dst; /** Destination (Remote) BR/EDR address */
 };
 
-/** Connection role (master or slave) */
+/** Connection role (central or peripheral) */
+#define BT_CONN_ROLE_MASTER __DEPRECATED_MACRO 0
+#define BT_CONN_ROLE_SLAVE __DEPRECATED_MACRO 1
+
 enum {
-	BT_CONN_ROLE_MASTER,
-	BT_CONN_ROLE_SLAVE,
+	BT_CONN_ROLE_CENTRAL = 0,
+	BT_CONN_ROLE_PERIPH = 1,
```

Comment:
```suggestion
	BT_CONN_ROLE_PERIPHERAL = 1,
```

I think it's worth using the full name here

Reply:
I was expecting this :)
No big preference for me here, but let's run a quick poll @cvinayak @joerchan @alwa-nordic @asbjornsabo @Vudentz @jhedberg @LingaoM @trond-snekvik @thoh-ot @kruithofa

Reply:
If we use the full name for the central, we can do so for the peripheral as well.

Reply:
I'd use the full name to be consistent with "central"

Reply:
Peripheral it is.

BT Core TS uses "CEN" and "PER" abbreviations, but Central and Peripheral is better in API

Reply:
Thanks for all the feedback. I have replaced it with the full word.